config/webpack/terra-dev-site.webpack.config.js

```diff
@@ -1,32 +1,92 @@
 
 const path = require('path');
 const HtmlWebpackPlugin = require('html-webpack-plugin');
+const generateAppConfig = require('../../scripts/generate-app-config/generateAppConfig');
+const loadDefaultSiteConfig = require('../../scripts/generate-app-config/loadDefaultSiteConfig');
 const fs = require('fs');
 
-const isFile = filePath => (fs.existsSync(filePath) && !fs.lstatSync(filePath).isDirectory());
+/**
+* Add's an alias and a 'source' alias if not in prod mode and hot reloading is enabled.
+*/
+const addAlias = (acc, name, location, hotReloading, production) => {
+  acc[name] = path.join(location);
+  if (!production && hotReloading.enabled) {
+    acc[`${name}/${hotReloading.dist}`] = `${name}/${hotReloading.source}`;
```

Comment:
I think path.join is in the wrong place. Should be like?:
```
acc[name] = location;
if (!production && hotReloading.enabled) {
    acc[path.join(name, hotReloading.dist)] = path.join(name, hotReloading.source);
```

Reply:
So the fun wth nested object config,  how do we want to fail if `hotReloading.dist` or `hotReloading. source` is undefined? Do we want to provide a fallback. It appear we are only doing aliasing on the default config and anyone providing their own site config will need to manual do this?